scala/build/Dockerfile.latest

```
@@ -0,0 +1,87 @@
+    curl -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose && \
+    chmod +x /usr/local/bin/docker-compose
+
+# Setup application directory
+RUN mkdir -p /home/qa/Downloads /app /home/qa/code && \
+chown -R 1000:1000 /home/qa
+
+# Set work directory
+WORKDIR /home/qa/code
+
+# Install Node & Yarn dependencies
+RUN yarn install && \
```

Hadolint

[info] `yarn cache clean` missing after `yarn install` was run.

(DL3060)